File: app-java/src/main/java/com/example/app_java/Models/Game.java
package com.example.app_java.Models;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;

public class Game {
    @JsonProperty("handle")
    private final String handle;

    @JsonProperty("turns_taken")
    private final int turnsTaken;

    @JsonProperty("time_taken")
    private final int timeTaken;

    @JsonProperty("cards")
    private final List<Card> cards;

    public Game(String handle, int turnsTaken, int timeTaken, List<Card> cards) {
        this.handle = handle;
        this.turnsTaken = turnsTaken;
        this.timeTaken = timeTaken;
        this.cards = cards;
    }

    // Getters
    public String getHandle() {
        return handle;
    }

    public int getTurnsTaken() {
        return turnsTaken;
    }

    public int getTimeTaken() {
        return timeTaken;
    }

    public List<Card> getCards() {
        return cards;
    }
}

File: complete-app/app/Models/Game.cs
using System.Text.Json.Serialization;

namespace app.Models
{
    public class Game
    {
        [JsonPropertyName("handle")]
        public required string Handle { get; set; }

        [JsonPropertyName("turns_taken")]
        public int TurnsTaken { get; set; }

        [JsonPropertyName("time_taken")]
        public int TimeTaken { get; set; }

        [JsonPropertyName("cards")]
        public List<Card> Cards { get; set; } = new();

        public int GameId { get; set; }

        public ICollection<GameCard> GameCards { get; set; } = [];

    }
}